tests/unit/components/Globe.spec.ts

```diff
@@ -0,0 +1,228 @@
+import Globe from "@/components/Globe.vue";
+import { mockNuxtImport, mountSuspended } from "@nuxt/test-utils/runtime";
+import { waitFor } from "@testing-library/vue";
+import { globeParameter, mockedMetadata, mockPinia, selectParameters, updatableNumericParameter } from "~/tests/unit/mocks/mockPinia";
+
+// https://developer.mamezou-tech.com/en/blogs/2024/02/12/nuxt3-unit-testing-mock/
+const { mockRoute } = vi.hoisted(() => ({
+  mockRoute: vi.fn(),
+}));
+
+mockNuxtImport("useRoute", () => mockRoute);
+
+afterEach(() => {
+  mockRoute.mockReset();
+});
+
+// NB: These component tests perform assertions about the return value of setUpChart(), which I call
+// directly in the tests. This is a second-best kind of test, since it means we're testing a
+// second chart object, rather than the one that was created automatically by the component
+// setup script. I do this because I couldn't find a way to access that first chart object from here.
```

Comment:
It runs setupChart twice (in the case of running a test) - once on setup and once when we call it directly. (As confirmed by console logging)

What do you mean by setting it 'as a value'? When I access 'component.vm.chart' it is mysteriously undefined. (In fact, it's whatever we initialize 'chart' to, since component.vm.chart is 3 when we initialize chart to 3).

Anyway, I've now found that if I swap out 'mountSuspended' (@nuxt/test-utils) to 'mount' (@vue/test-utils) the problem goes away.